DSCResources/MSFT_xDSCWebService/MSFT_xDSCWebService.psm1

```diff
@@ -75,19 +78,17 @@ function Get-TargetResource
 
             $webBinding = Get-WebBinding -Name $EndpointName
 
-            # This is the 64 bit module
-            $certNativeModule = Get-WebConfigModulesSetting -WebConfigFullPath $webConfigFullPath -ModuleName "IISSelfSignedCertModule" 
-            if($certNativeModule)
+            $iisSelfSignedModuleName = "IISSelfSignedCertModule"
+            if ($Enable32BitAppOnWin64 -eq $true)
             {
-                $AcceptSelfSignedCertificates = $true
-            }           
-
-            # This is the 32 bit module
-            $certNativeModule = Get-WebConfigModulesSetting -WebConfigFullPath $webConfigFullPath -ModuleName "IISSelfSignedCertModule(32bit)" 
+                $iisSelfSignedModuleName = "IISSelfSignedCertModule(32bit)"
```

Comment:
How does this work if we are on a 32 bit system? Isn't it going to look for the 64 bit version now?